docs/docs/prototype-an-assistant.mdx

```diff
@@ -3,9 +3,12 @@ id: prototype-an-assistant
 sidebar_label: 'Rasa Playground'
 title: 'Rasa Playground'
 abstract: Learn the basics of building an assistant with Rasa Open Source with this interactive guide. You will be able to customize the assistant, talk to it and download the project so you can continue to build.
+hide_table_of_contents: true
+full_width: true
```

Comment:
it doesn't appear to work:

https://drive.google.com/file/d/1LouMzeZDJAuIje1Xa1KWoIc9aZ_QhGKA/view?usp=sharing

Reply:
That's because this change needs to be merged first: https://github.com/RasaHQ/tabula/pull/31

Reply:
👍